semgrep-core/src/parsing/pfff/Python_to_generic.ml

```diff
@@ -199,7 +199,8 @@ let rec expr env (x : expr) =
       |> G.e
   | InterpolatedString xs ->
       G.Call
-        ( G.IdSpecial (G.ConcatString G.FString, unsafe_fake "concat") |> G.e,
+        ( G.IdSpecial (G.ConcatString (G.FString "f"), unsafe_fake "concat")
```

Comment:
A comment to give context would be good here (Python strings are always `f"..."`, corresponding to Scala's `abc"..."`)

Reply:
done